README.md

```diff
@@ -55,7 +55,7 @@
   <a href="#badge">
     <img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square"></a>
   <a href="https://twitter.com/PrettierCode">
-    <img alt="Follow Prettier on Twitter" src="https://img.shields.io/twitter/follow/prettiercode.svg?label=follow+prettier&style=flat-square"></a>
+    <img alt="Follow Prettier on Twitter" src="https://img.shields.io/twitter/follow/prettiercode.svg"></a>
```

Comment:
```suggestion
    <img alt="Follow Prettier on Twitter" src="https://img.shields.io/badge/follow-@PrettierCode-9f9f9f.svg?style=flat-square"></a>
```

WDYT?

Reply:
Or

```
https://img.shields.io/badge/Follow-%40PrettierCode-9f9f9f?style=flat-square&logo=x
```

![](https://img.shields.io/badge/Follow-%40PrettierCode-9f9f9f?style=flat-square&logo=x)

Reply:
Or just the logo

```
https://img.shields.io/badge/-%40PrettierCode-9f9f9f?style=flat-square&logo=x&labelColor=000
```

![](https://img.shields.io/badge/-%40PrettierCode-9f9f9f?style=flat-square&logo=x&labelColor=000)

Reply:
I prefer just the logo pattern